datafusion/core/tests/sqllogictests/test_files/aggregate.slt

```diff
@@ -52,7 +52,7 @@ SELECT var_pop(c2) FROM aggregate_test_100
 query R
 SELECT var_pop(c6) FROM aggregate_test_100
 ----
-2.615633434202189e37
+26156334342021890000000000000000000000
```

Comment:
these differences are due the difference in `arrow::util::display::array_value_to_string` and the CSV writer. I think the changes are an improvement